#! /bin/sh /usr/share/dpatch/dpatch-run
## 0020_saslauthd_manpage.dpatch by  Thijs Kinkhorst <thijs@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct typos in saslauthd man page.

@DPATCH@
diff -urNad trunk~/saslauthd/saslauthd.8 trunk/saslauthd/saslauthd.8
--- trunk~/saslauthd/saslauthd.8	2006-06-04 12:26:20.000000000 +0300
+++ trunk/saslauthd/saslauthd.8	2008-02-18 13:31:32.000000000 +0200
@@ -37,7 +37,7 @@
 
      [1m-H [4m[22mhostname[0m
              The remote host to be contacted by the rimap authentication mech-
-             anism. (Depricated, use -O instead)
+             anism. (Deprecated, use -O instead)
 
      [1m-m [4m[22mpath[0m
              Use [4mpath[24m as the pathname to the named socket to listen on for
@@ -51,7 +51,7 @@
              Use [4mthreads[24m processes for responding to authentication queries.
              (default: 5)  A value of zero will indicate that saslauthd should
              fork an individual process for each connection.  This can solve
-             leaks that occur in some deployments..
+             leaks that occur in some deployments.
 
      [1m-s [4m[22msize[0m
              Use [4msize[24m as the table size of the hash table (in kilobytes)
@@ -64,7 +64,7 @@
 
      [1m-h      [22mShow usage information
 
-     [1m-c      [22mEnable cacheing of authentication credentials
+     [1m-c      [22mEnable caching of authentication credentials
 
      [1m-l      [22mDisable the use of a lock file for controlling access to
              accept().
@@ -72,7 +72,7 @@
      [1m-r      [22mCombine the realm with the login (with an '@' sign in between).
              e.g.  login: "foo" realm: "bar" will get passed as login:
              "foo@bar".  Note that the realm will still be passed, which may
-             lead to unexpected behavior.
+             lead to unexpected behaviour.
 
      [1m-v      [22mPrint the version number and available authentication mechanisms
              on standard error, then exit.
@@ -80,7 +80,7 @@
      [1m-d      [22mDebugging mode.
 
    [1mLogging[0m
-     [1msaslauthd [22mlogs it's activities via [1msyslogd [22musing the LOG_AUTH facility.
+     [1msaslauthd [22mlogs its activities via [1msyslogd [22musing the LOG_AUTH facility.
 
 [1mAUTHENTICATION MECHANISMS[0m
      [1msaslauthd [22msupports one or more "authentication mechanisms", dependent
@@ -96,7 +96,7 @@
 
                 Authenticate using the [1mgetpwent[22m() library function. Typically
                 this authenticates against the local password file. See your
-                systems getpwent(3) man page for details.
+                system's getpwent(3) man page for details.
 
      kerberos4  [4m(All[24m [4mplatforms)[0m
 
@@ -145,7 +145,7 @@
      sasldb     [4m(All[24m [4mplatforms)[0m
 
                 Authenticate against the SASL authentication database.  Note
-                that this is probabally not what you want to be using, and is
+                that this is probably not what you want to be using, and is
                 even disabled at compile-time by default.  If you want to use
                 sasldb with the SASL library, you probably want to use the
                 pwcheck_method of "auxprop" along with the sasldb auxprop plu-
diff -urNad trunk~/saslauthd/saslauthd.mdoc trunk/saslauthd/saslauthd.mdoc
--- trunk~/saslauthd/saslauthd.mdoc	2006-06-04 12:26:20.000000000 +0300
+++ trunk/saslauthd/saslauthd.mdoc	2008-02-18 13:33:12.000000000 +0200
@@ -64,7 +64,7 @@
 .It Fl H Ar hostname
 The remote host to be contacted by the
 .Li rimap
-authentication mechanism. (Depricated, use -O instead)
+authentication mechanism. (Deprecated, use -O instead)
 .It Fl m Ar path
 Use
 .Ar path
@@ -79,7 +79,7 @@
 processes for responding to authentication queries. (default: 5)  A
 value of zero will indicate that saslauthd should fork an individual
 process for each connection.  This can solve leaks that occur in some
-deployments..
+deployments.
 .It Fl s Ar size
 Use
 .Ar size
@@ -93,13 +93,13 @@
 .It Fl h
 Show usage information
 .It Fl c
-Enable cacheing of authentication credentials
+Enable caching of authentication credentials
 .It Fl l
 Disable the use of a lock file for controlling access to accept().
 .It Fl r
 Combine the realm with the login (with an '@' sign in between).  e.g.
 login: "foo" realm: "bar" will get passed as login: "foo@bar".  Note that
-the realm will still be passed, which may lead to unexpected behavior.
+the realm will still be passed, which may lead to unexpected behaviour.
 .It Fl v
 Print the version number and available authentication
 mechanisms on standard error, then exit.
@@ -108,7 +108,7 @@
 .El
 .Ss Logging
 .Nm
-logs it's activities via
+logs its activities via
 .Nm syslogd
 using the
 .Dv LOG_AUTH
@@ -132,7 +132,7 @@
 Authenticate using the
 .Fn getpwent
 library function. Typically this authenticates against the
-local password file. See your systems
+local password file. See your system's
 .Xr getpwent 3
 man page for details.
 .It Li kerberos4
@@ -207,8 +207,8 @@
 .Em (All platforms)
 .Pp
 Authenticate against the
-SASL authentication database.  Note that this is probabally not what you
-want to be using, and is even disabled at compile-time by default.
+SASL authentication database.  Note that this is probably not what you
+want to use, and is even disabled at compile-time by default.
 If you want to use sasldb with the SASL library, you probably want to
 use the pwcheck_method of "auxprop" along with the sasldb auxprop plugin
 instead.
